server/controllers/finance/reports/balance_sheet/report.handlebars

```diff
@@ -0,0 +1,138 @@
+
+<body>
+
+  {{> header}}
+
+  <!-- body  -->
+  <div class="row">
+    <div class="col-xs-12">
+
+      <!-- page title  -->
+      <h2 class="text-center text-uppercase">
+        {{translate 'REPORT.BALANCE_SHEET.TITLE'}}
+      </h2>
+
+      <h4 class="text-center">
+        {{#if session.dateFrom}}
+          {{date session.dateFrom}} - {{date session.dateTo}}
+        {{/if}}
+        {{#if session.date}}
+          {{date session.date}}
+        {{/if}}
+      </h4>
+
+      <table class="report-condensed-table">
+        <thead>
+          <tr>
+            <th style="width: 70%;"></th>
+            <th style="width: 15%;" class="text-center">{{translate 'REPORT.BALANCE_SHEET.DEBTOR_SOLD'}}</th>
+            <th style="width: 15%;" class="text-center">{{translate 'REPORT.BALANCE_SHEET.CREDITOR_SOLD'}}</th>
+          </tr>
+        </thead>
+      </table>
+
+      {{!-- assets --}}
+      <h4 style="margin: 2px;"><strong>{{translate "ACCOUNT.TYPES.ASSET"}}</strong></h4>
+      <table class="report-condensed-table">
+        <tbody>
+          {{#each assets.accounts as |account|}}
+            {{#if account.balance}}
+            <tr>
+              <td style="width: 70%;" class="text-left"><b>{{ account.number }}</b> - {{ account.label }}</td>
+              <td style="width: 15%;" class="text-right">{{#gt account.balance 0}}{{currency account.balance metadata.enterprise.currency_id}}{{/gt}}</td>
+              <td style="width: 15%;" class="text-right">{{#lt account.balance 0}}{{currency (multiply account.balance -1) metadata.enterprise.currency_id}}{{/lt}}</td>
+            </tr>
+            {{/if}}
+          {{/each}}
+        </tbody>
+        <tfoot>
+          <tr class="text-right" style="font-weight: bold; background-color: #efefef;">
+            <th>{{translate "FORM.LABELS.TOTAL"}} {{translate "ACCOUNT.TYPES.ASSET"}}</th>
+            <th class="text-right">{{#gt assets.totals.balance 0}}{{currency assets.totals.balance metadata.enterprise.currency_id}}{{/gt}}</th>
+            <th class="text-right">{{#lt assets.totals.balance 0}}{{currency (multiply assets.totals.balance -1) metadata.enterprise.currency_id}}{{/lt}}</th>
+          </tr>
+        </tfoot>
+      </table>
+
+      {{!-- liabilities --}}
+      <h4 style="margin: 2px;"><strong>{{translate "ACCOUNT.TYPES.LIABILITY"}}</strong></h4>
+      <table class="report-condensed-table">
+        <tbody>
+          {{#each liabilities.accounts as |account|}}
+            {{#if account.balance}}
+            <tr class="text-right">
+              <td style="width: 70%;"  class="text-left"><b>{{ account.number }}</b> - {{ account.label }}</td>
+              <td style="width: 15%;"  class="text-right">{{#gt account.balance 0}}{{currency account.balance metadata.enterprise.currency_id}}{{/gt}}</td>
+              <td style="width: 15%;"  class="text-right">{{#lt account.balance 0}}{{currency (multiply account.balance -1) metadata.enterprise.currency_id}}{{/lt}}</td>
```

Comment:
It is best to reduce the amount logic that templates have to perform - it is much easier for another developer to understand what is happening if the display layer (this HTML template) does not manipulate the values directly. 

This value should be calculated in the controller.